crates/editor/src/selections_collection.rs

```diff
@@ -426,44 +426,24 @@ impl<'a> MutableSelectionsCollection<'a> {
         self.selections_changed = true;
     }
 
-    pub(crate) fn set_pending_display_range(
```

Comment:
This method has a single appearance in mouse_context_menu. So, I'm simplifying what it does.